packages/dataviews/src/view-table.js

```diff
@@ -93,26 +94,34 @@ function HeaderMenu( { field, view, onChangeView } ) {
 								const isChecked =
 									isSorted &&
 									view.sort.direction === direction;
+
+								const value = `${ field.id }-${ direction }`;
+
 								return (
-									<DropdownMenuRadioItemCustom
-										key={ direction }
-										name={ `view-table-sort-${ field.id }` }
-										value={ direction }
+									<DropdownMenuRadioItem
+										key={ value }
+										// All sorting radio items share the same name, so that
+										// selecting a sorting option automatically deselects the
+										// previously selected one, even if it is displayed in
+										// another submenu. The field and direction are passed via
+										// the `value` prop.
```

Comment:
We don't use the `value` right now, so this part of the comment could be removed, right?

Reply:
The `value` is required by radio items, and is necessary to distinguish each radio item from the other in the internal logic (since the all share the same name)

Reply:
Sure, just referring to `The field and direction are passed via the value prop.` part of comment. Is this part relevant?

Reply:
In my mind, it is — it complements the fact that `name` is very generic and doesn't include field and/or direction